communication/mavlink_waypoint_handler.cpp

```diff
@@ -1767,13 +1756,6 @@ Mavlink_waypoint_handler::Mavlink_waypoint_handler(Position_estimation& position
     callback.module_struct  = (Mavlink_message_handler::handling_module_struct_t) this;
     init_success &= message_handler.add_msg_callback(&callback);
 
-    callback.message_id     = MAVLINK_MSG_ID_SET_GPS_GLOBAL_ORIGIN; // 48
```

Comment:
as discuss you shouldn't remove it but tjust comment out that part, and deal with it in the open issue or PR